tests/unit/Commands/SyncWoocommerceAttributesTest.php

```diff
@@ -203,4 +199,41 @@ public function assertReservedAttributes($original_attribute_data, string $messa
 
         return $wc_attribute;
     }
+
+    protected function buildExpectedResults($original_attribute_data): array
```

Comment:
maybe better to just put the expected result in json ? I mean you only use `id,name,label` from it and this code looks like the code which is beeing tested